.github/workflows/test-and-release.yml

```diff
@@ -0,0 +1,72 @@
+name: 'Test and release snyk-to-html'
+
+on:
+  push
+
+jobs:
+  lint: 
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: npm config get cache
+        id: npm-cache-dir
+        run: echo "::set-output name=dir::$(npm config get cache)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.npm-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-node-14-${{ hashFiles('**/package.json') }}
+          restore-keys: ${{ runner.os }}-node-14-
+      - run: npm ci
+      - run: npm run lint
+
+  test:
```

Comment:
This needs a `needs: ['lint']` so it runs after lint.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
done